azavea/temperate
# Hide assess button for initiated risk assessments

## Overview

If a user has started an assessment for a risk, the assess button should not be shown when they return to the assessment overview page. A progress bar will be added in it's place in #623

### Demo

![feb-21-2018 15-31-45](https://user-images.githubusercontent.com/1042475/36503824-74377332-171c-11e8-9fa2-75ee4bd5bb5d.gif)

## Testing Instructions

- Visit the assessment overview page and add a new hazard.
- Click the "Assess" button for one of the risks.
- Proceed through a few of the steps, and then click "Finish later".
- When you are returned to the assessment overview page, verify that the "Assess" button is no longer displayed for the risk assessment you started.

Closes #621
